mappymatch/utils/plot.py

```diff
@@ -178,3 +192,91 @@ def plot_map(tmap: NxMap, m=None):
         ).add_to(m)
 
     return m
+
+
+def plot_match_distances(coord_df):
```

Comment:
I think the function signature would be more flexible if we asked for the `MatchResult` type here (which is just a list of `Match` objects right now). Then, we could just pull the distances directly from that and not require the user to build a dataframe before using this function. Something like:

```python
def plot_match_distances(matches: MatchResult):
```

Reply:
I can do that. I added 
`from mappymatch.matchers.matcher_interface import MatchResult `
to include that type in the plot.py file.